ament_copyright/ament_copyright/parser.py

```diff
@@ -54,14 +55,6 @@ def read(self):
     def parse(self):
         raise NotImplemented()
 
-    def identify_license(self, content, license_part):
-        for name, license in get_licenses().items():
-            if content is not None and getattr(license, license_part) == content:
-                self.license_identifier = name
-                break
-        else:
-            self.license_identifier = UNKNOWN_IDENTIFIER
-
```

Comment:
I need to override identify_license() because the SourceDescriptor version of it wants to iterate over the self.copyrights list, which only exists in the SourceDescriptor subclass.  I can restore the generic version to FileDescriptor and override it in SourceDescriptor, but it seems like overkill since it is such a simple function.